problems/totalNumberOfStudents.js

```diff
@@ -12,7 +12,13 @@
 */
 
 function totalNumberOfStudents() {
+    for( arr in totalNumberOfStudents[" "] [arr]) {
+        arr.push(totalNumberOfStudents[" "] [arr])
+    }
```

Comment:
Param missing, and this wouldn't work. Please review objects and how to use objects with for in's